components/tikv_alloc/Cargo.toml

```diff
@@ -6,7 +6,8 @@ authors = ["Brian Anderson <andersrb@gmail.com>"]
 publish = false
 
 [features]
-default = ["jemalloc"]
+default = []
```

Comment:
default is what now? still jemalloc?

Reply:
@siddontang  Tikv's default is `jemalloc`.  if compile `tikv-alloc` separately，you need to specify features.

Reply:
@siddontang Because I can't give `tikv_alloc` a default features,  This is to avoid compiling jemalloc when using other features.

Reply:
As it stands in this PR, the crate requires the user to specify a malloc or it will not compile.